dolphinscheduler-server/src/main/java/org/apache/dolphinscheduler/server/builder/TaskExecutionContextBuilder.java

```diff
@@ -55,11 +55,12 @@ public TaskExecutionContextBuilder buildTaskInstanceRelatedInfo(TaskInstance tas
 
     public TaskExecutionContextBuilder buildTaskDefinitionRelatedInfo(TaskDefinition taskDefinition) {
         int timeoutSeconds = taskDefinition.getTimeout() * SEC_2_MINUTES_TIME_UNIT;
-        if (timeoutSeconds >= Integer.MAX_VALUE) {
+        if (timeoutSeconds == 0) {
```

Comment:
if a==0 then  a=max_value？

Reply:
I will fix it, according to timeout notify strategy